### PR TITLE
Use node-riffraff-artefact to deploy s3watcher

### PR DESCRIPTION
**Pull request by kenoir**

### CODE REVIEW

File: s3watcher/lambda/cloudformation.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "file_left_intentionally": "blank"
```
Comment (akash1810) [inline s3watcher/lambda/cloudformation.json]: :confused:

Comment (kenoir) [inline s3watcher/lambda/cloudformation.json]: `node-riffraff-artefact` currently fails without this file. 

Probably should PR another update.

**Comment by kenoir** [comment]
Will also work in TC (and simplify build config).

File: s3watcher/build.sh [hunk 2]
```diff
@@ -12,28 +12,14 @@ if [ -z $PROD_FUNC_NAME ]; then
...
-mkdir -p target/packages/s3-watcher
-
-cp lambda/s3-watcher.zip target/packages/lambda/lambda.zip
-cp lambda/s3-watcher.zip target/packages/s3-watcher/s3-watcher.zip
+cat deploy.json \
+    | jq ".packages.s3watcher.data.functions.TEST.name |= \"$TEST_FUNC_NAME\" | .packages.s3watcher.data.functions.PROD.name |= \"$PROD_FUNC_NAME\"" \
+    > lambda/deploy.json
 
-rm lambda/s3-watcher.zip
+export VERBOSE=true
```
Comment (akash1810) [inline s3watcher/build.sh]: what's this used for?

Comment (kenoir) [inline s3watcher/build.sh]: It makes the `node-riffraff-artefact` lib log verbose which I think is useful in CI (immediately pinpoint problems with builds).

**Comment by akash1810** [comment]
:+1:

**Comment by kenoir** [comment]
This build breaks on  TC at the moment with this error:

`Failed to create zip with: Error: stdout maxBuffer exceeded.`

This is related to an issue with https://github.com/guardian/node-riffraff-artefact building the zip. LIkely the buffer size values seen here: https://github.com/guardian/node-riffraff-artefact/blob/master/src/lib/util.js#L60 need to be configurable.

**Comment by kenoir** [comment]
^ fixed

**Comment by kenoir** [comment]
For those interested: Currently the TC build is pretending to be Jenkins by being sneaky with environment variables!

**Comment by akash1810** [comment]
> For those interested: Currently the TC build is pretending to be Jenkins by being sneaky with environment variables!

`export ENV=jenkins` ? :stuck_out_tongue_closed_eyes:

**Pull request merged**